scripting/get5/natives.sp

```diff
@@ -251,3 +252,10 @@ public int Native_GetMatchStats(Handle plugin, int numParams) {
     return view_as<int>(true);
   }
 }
+
+public int Native_CreateGet5HTTPRequest(Handle plugin, int numParams) {
```

Comment:
We need to add a check here for the SteamWorks library, but we can't run it all the time as it's expensive, so we have to have a boolean/int check. Is there a specific reason this returns an `int` and not a `Handle?` Must a Native return an integer? If yes, just cast `INVALID_HANDLE` as int here.

```c
static int hasSteamWorks = 0;
if (hasSteamWorks != 1 && (hasSteamWorks == -1 || !LibraryExists("SteamWorks"))) {
  LogError("The SteamWorks extension is required in order to make HTTP requests with Get5.");
  hasSteamWorks = -1;
  return INVALID_HANDLE;
}
hasSteamWorks = 1;
```